pages/calculator/[slug].tsx

```diff
@@ -101,7 +103,7 @@ export default function CalculatorSlugRoute({ page, calculatorConfig }: StaticCa
                 textAlign="center"
                 id="legal-disclaimer-container"
               >
-                <Typography variant="caption" data-cy="legal-disclaimer" sx={{ fontWeight: 'light' }}>
+                <Typography variant="caption" sx={{ fontWeight: 'light' }}>
```

Comment:
@aitzeng can you please update this so that you are not accidentally deleting the `data-cy`

Reply:
Readded it